### PR TITLE
CLI fix

### PR DESCRIPTION
**Pull request by hanpeter**
Removing CLI arguments from the `krux_sqs.cli.Application` so it can be inherited.

### CODE REVIEW

**Comment by dpricha89** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -32,4 +32,4 @@ if __name__ == '__main__':
 
 ```
 
-As long as you get an instance of `krux_boto.boto.Boto3`, the rest are the same. Refer to `krux_boto` module's [README](../krux_boto/README.md) on various ways to instanciate the class.
+As long as you get an instance of `krux_boto.boto.Boto3`, the rest are the same. Refer to `krux_boto` module's [README](https://github.com/krux/python-krux-boto/blob/master/README.md) on various ways to instanciate the class.
```
Comment (dcrampton-sfdc) [inline README.md:35]: Yay for absolute link!

**Comment by dcrampton-sfdc** [comment]
LGTM. Why does it need to be inherited?

File: README.md (hunk 1 above)
Comment (hanpeter) [inline README.md:35]: Yeah, I would've preferred a relative links, but I don't think I can go into a different repo using relative links. The only way that I can get working is very hacky.

**Comment by hanpeter** [comment]
@dcrampton: It doesn't 'need' to be inherited. But we inherit our CLI classes a lot.

Instead of writing this:

``` python
class Application(krux.cli.Application):
    def __init__(self):
        self.boto = Boto(access_key='ABC', secret_key='XYZ', logger=self.logger, stats=self.stats)
        self.sqs = Sqs(boto=self.boto, logger=self.logger, stats=self.stats)
    def add_cli_arguments(self, parser):
        add_sqs_cli_arguments(parser)
    def run(self):
        self.sqs.go()
```

You can just write:

``` python
class Application(krux_sqs.cli.Application):
    def run(self):
        self.sqs.go()
```

So it is much more convenient way to write CLI applications. It means there are some more management burden on the library itself, but the potential gain of reusable code is greater.

**Pull request merged**